### PR TITLE
[Merged by Bors] - chore: distribute fun_prop tagging of measurability lemmas

### PR DESCRIPTION
**Pull request by grunweg**
into the respective files: essentially dissolves the `FunProp/Measurable` and `AEMeasurable` files.
(The latter is kept to note two missing statements. It is not imported anywhere any more.)

This is a pre-requisite for #13781, so fun_prop can be used without breaking existing the `assert_not_exists` statements.

---

Aside. The module docstring for Constructions/BorelSpace/Basic is outdated, `Measurable.add` is not mentioned there any more.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by lecopivo** [comment]
Why didn't you redistribute all the attributes? And didn't remove the file [Mathlib/Tactic/FunProp/Measurable.lean](https://github.com/leanprover-community/mathlib4/pull/13869/files#diff-f02e8bf47051030ae095d3ce32f934f8cae8738f7dd92460045677c6852f644e) ? 

Otherwise the PR is good with me.

**Comment by grunweg** [comment]
Thanks for taking a look! This is the smallest change I needed to make the dependent PR go through. But I'm happy to redistribute the remaining tags also... this might happen only later this week.

**Comment by grunweg** [comment]
I moved the remaining attributes in `Measurable` and `AEMeasurable`; this is ready for review again.

File: Mathlib/Tactic/FunProp/AEMeasurable.lean [hunk 1]
```diff
@@ -4,64 +4,15 @@ Released under Apache 2.0 license as described in the file LICENSE.
...
 
-
-section missing
-
-open MeasureTheory
-
-variable {ι α β γ δ R : Type*} {m0 : MeasurableSpace α} [MeasurableSpace β] [MeasurableSpace γ]
-  [MeasurableSpace δ] {f g : α → β} {μ ν : Measure α}
-
-theorem AEMeasurable.comp_aemeasurable' {f : α → δ} {g : δ → β} (hg : AEMeasurable g (μ.map f))
```
Comment (grunweg) [inline Mathlib/Tactic/FunProp/AEMeasurable.lean:23]: Moved this one to `Mathlib/MeasureTheory/Measure/AEMeasurable.lean`. Happy to split that into a separate PR, if preferred.

**Comment by lecopivo** [comment]
Looks good to me!

File: Mathlib/Tactic/FunProp/AEMeasurable.lean [hunk 2]
```diff
@@ -4,64 +4,15 @@ Released under Apache 2.0 license as described in the file LICENSE.
...
-  AEMeasurable.inv
-  AEMeasurable.smul
-
--- transitions
-attribute [fun_prop]
-  AEMeasurable.mono'
-  Measurable.aemeasurable
-
+-- lambda rules: these two are currently missing
+-- attribute [fun_prop] AEMeasurable_apply AEMeasurable_pi
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/FunProp/AEMeasurable.lean:15]: Can these comments not go next to these lemmas?

Comment (grunweg) [reply Mathlib/Tactic/FunProp/AEMeasurable.lean:15]: For these two lemmas, that's difficult as the *lemmas* are missing. I moved the other two comments, though - good point, thanks!

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by semorrison** [comment]
bors merge
